src/unum/unum.c

```diff
@@ -628,10 +636,16 @@ static int do_config_char(char opt_long, char *optarg)
             status = util_set_opmode(optarg);
             if(status < 0) {
                 // Give status offset to avoid overlap with other errors here
-                status -= 10;
+                status = -10;
                 break;
             }
             break;
+        case 'L':
+            if(!util_path_exists(optarg)) {
+                status = -11;
```

Comment:
I think you misunderstood how this works. 
Please make this line return -3 and the "default" case return -4,
then `case: 'L'` is better be above `case: 'o'` and `case: 'o'` should be returning the `status` offset to -10 (not just -10).

Reply:
@dob71 I have corrected the error codes in further check-ins. Now I have moved the case:'L' above case:'o'